core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -557,7 +558,27 @@ case object ModelInsights {
             case _ => None
           }
           val keptIndex = indexInToIndexKept.get(h.index)
-
+          val featureStd = getIfExists(h.index, s.featuresStatistics.variance).getOrElse(1.0)
+          val sparkFtrContrib = keptIndex
+            .map(i => contributions.map(_.applyOrElse(i, (_: Int) => 0.0))).getOrElse(Seq.empty)
+          val LRStandardization = checkLRStandardization(model).getOrElse(false)
+
+          val labelStd = label.distribution match {
+            case Some(Continuous(_, _, _, variance)) => math.sqrt(variance)
+            case Some(Discrete(domain, prob)) =>
+              val (weighted, sqweighted) = (domain zip prob).foldLeft((0.0, 0.0)) {
+                case ((weightSum, sqweightSum), (d, p)) =>
+                  val floatD = d.toDouble
+                  val weight = floatD * p
+                  val sqweight = floatD * weight
+                  (weightSum + weight, sqweightSum + sqweight)
+              }
+              sqweighted - weighted
+            case Some(d) => throw new RuntimeException("Unsupported distribution type for the label")
+            case None => throw new RuntimeException("Label does not exist, please check your data")
+          }
+          if (labelStd == 0) throw new RuntimeException("The standard deviation of the label is zero, " +
+            "so the coefficients and intercepts of the model will be zeros, training is not needed.\"")
```

Comment:
In general we have tried not to have the model insights throw exceptions - it is kind of a rude place to fail. Please have it log a warning and put in a graceful empty default value